tests/cnf/ran/talm/tests/talm-batching.go

```diff
@@ -199,8 +199,10 @@ var _ = Describe("TALM Batching Tests", Label(tsparams.LabelBatchingTestCases),
 				Spoke2APIClient, tsparams.CatalogSourceName, tsparams.TemporaryNamespace).Exists()
 			Expect(catSrcExistsOnSpoke2).To(BeFalse(), "Catalog source exists on spoke 2")
 		})
-
-		It("should continue the CGU when the first batch fails with the Continue batch timeout action", func() {
+		// 74753 upgrade failure of first batch would not affect second batch
+		It("should continue the CGU when the first batch fails with the Continue batch timeout"+
+			"action", reportxml.ID("74753"), func() {
```

Comment:
@klaskosk @josclark42 fyi seems both lint and IDE are happy with this format.

Reply:
I'll have to keep this in mind for other cases so it doesn't indent, thanks